### PR TITLE
SoftwareTeam.KojiPolicyWriting

### PR DESCRIPTION
**Pull request by matyasselmeci**
https://twiki.opensciencegrid.org/bin/view/SoftwareTeam/KojiPolicyWriting

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Add migration header to TWiki document

### CODE REVIEW

File: docs/infrastructure/koji-policy-writing.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+**source:**
+
+-   `hub/kojihub.py:pkglist_unblock`   
+    `unblock-pkg` command
+
+**policy data:** same as above, except `action` is 'unblock'
+
+#### tag
+
+NOTE: RootExports is the class containing functions exported via XMLRPC. In general, each function corresponds to a koji task.
```
Comment (brianhlin) [inline docs/infrastructure/koji-policy-writing.md]: Could we make this an admonition?

```
!!! note
    RootExports...
```

**Comment by matyasselmeci** [comment]
Done.

**Comment by brianhlin** [comment]
Could you fix the conflicts with master?

**Comment by matyasselmeci** [comment]
Conflicts fixed.

**Pull request merged**